docs/_data/portland-2024-config.yaml

```diff
@@ -275,3 +275,4 @@ flaghasboat: False
 flaghaswritingday: True
 flaghasjobfair: True
 flaghasbadgeflair: True
+flaghaslightningtalks: True
```

Comment:
I don't think this exists?

Reply:
Yea, need to add it. Katie suggested adding all the events to the front page, which I think is a :100: idea, and done here.

Reply:
Expanding the front page with more icons looks great, though I suspect the new additions need redesigning, but we don't need to block.